src/chrome/content/rules/administrator.de.xml

```diff
@@ -0,0 +1,8 @@
+<ruleset name="administrator.de">
+<target host="*.administrator.de" />
+<rule from="^http:" to="https:" />
+<test url="http://www.administrator.de/" />
```

Comment:
My apologies, I have mislead you. Just simplify to:
```
<ruleset name="administrator.de">
<target host="*.administrator.de" />
<rule from="^http:" to="https:" />
</ruleset>
```
But I personally still prefer to list all subdomains instead of `*.administrator.de`. I understand your argument that * can cover future subdomain. But in my experience, it often causes mixed content blocking (MCB) issue as some subdomains don't serve HTTPS. We never know whether new subdomain can support HTTPS. By listing all subdomains, we know that those subdomains *work*.